<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lab5</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,0">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Количество вершин</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="populationSizeSpinBox">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="solveButton">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Решить</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="regenerateButton">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Повторная генерация графа</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="numVerticesSpinBox">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="mutationRateSpinBox">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Вероятность мутации</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Размер популяции</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Количество поколений</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="numGenerationSpinBox">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
